source/iterator/snapshot.go

```diff
@@ -0,0 +1,246 @@
+// Copyright © 2022 Meroxa, Inc & Yalantis.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iterator
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/huandu/go-sqlbuilder"
+	"github.com/jmoiron/sqlx"
+
+	"github.com/conduitio-labs/conduit-connector-db2/coltypes"
+	"github.com/conduitio-labs/conduit-connector-db2/source/position"
+)
+
+// snapshotIterator - iterator which get snapshot data.
+// A "snapshot" is the state of a table data at a particular point in time when connector starts work.
+// First time when the snapshot iterator starts work, it is get max value from `orderingColumn` and saves
```

Comment:
Nitpicks: "First time" -> "The first time" and  "it is get max value" -> "it gets"

Reply:
Thanks, fixed